malaria24/ona/admin.py

```diff
@@ -57,10 +58,29 @@ class ReportedCaseAdmin(admin.ModelAdmin):
                     'gender',
                     'facility_code',
                     'landmark',
-                    'create_date_time')
+                    'create_date_time',
+                    'ehp_report')
```

Comment:
Good catch, should be `ehp_report_link` which is defined on the `ReportedCaseAdmin` class.